src/oracle/monitor.js

```diff
@@ -165,89 +172,86 @@ const updateMetrics = (oracleName, oracle, feed, value, id, actualPrice, lastRou
  * @param {*} value balance value to set
  */
 const updateBalanceMetrics = (oracleName, oracle, brand, value) => {
-    oracleBalance.labels(oracleName, oracle, brand).set(value)
-}
+  oracleBalance.labels(oracleName, oracle, brand).set(value);
+};
 
 /**
  * Function to query price for feed
  * @param {*} feed feed like 'BRAND_IN-BRAND_OUT'
  * @returns the price of the feed
  */
 const queryPrice = async (feed) => {
-    try {
-        const capDataStr = await vstorage.readLatest(
-            `published.priceFeed.${feed}_price_feed`,
-        );
-
-        //parse the value
-        var capData = JSON.parse(JSON.parse(capDataStr).value)
-        capData = JSON.parse(capData.values[0])
-        //replace any extra characters
-        capData = JSON.parse(capData.body.replaceAll("\\", ""))
-
-        //get the latest price by dividing amountOut by amountIn
-        var latestPrice = Number(capData.amountOut.value.digits) / Number(capData.amountIn.value.digits)
-        amountsIn[feed] = Number(capData.amountIn.value.digits)
-
-        console.log(feed + " Price Query: " + String(latestPrice))
-        actualPriceGauge.labels(feed).set(latestPrice)
-        return latestPrice
-
-    }
-    catch (err) {
-        console.log("Price could not be obtained")
-        return 0
-    }
+  try {
+    const capDataStr = await vstorage.readLatest(
+      `published.priceFeed.${feed}_price_feed`
+    );
 
-}
+    //parse the value
+    let capData = JSON.parse(JSON.parse(capDataStr).value);
+    capData = JSON.parse(capData.values[0]);
+    // Replace any extra characters
```

Comment:
```suggestion
    capData = JSON.parse(capData.values[0]);
    
    // Replace any extra characters
```

Reply:
Fixed in https://github.com/jacquesvcritien/agoric-cl-middleware/commit/bab17ace6325bd7bc9bd8eac8157715d976ffd97